BoxContentSDK/BoxContentSDK/Operations/BOXAPIDataOperation.m

```diff
@@ -228,39 +223,8 @@ - (void)close
 - (void)abortWithError:(NSError *)error
 {
     [self close];
-    [self.connection cancel];
+    self.error = error;
```

Comment:
previously we call [self connection:self.connection didFailWithError:error]; when abort, and inside didFailWithError:, we check if error!=nil before setting self.error